src/nvim/fileio.c

```diff
@@ -4082,19 +4091,21 @@ static int buf_write_bytes(struct bw_info *ip)
   return (wlen < len) ? FAIL : OK;
 }
 
-/*
- * Convert a Unicode character to bytes.
- * Return TRUE for an error, FALSE when it's OK.
- */
-static int 
-ucs2bytes (
-    unsigned c,                     /* in: character */
-    char_u **pp,               /* in/out: pointer to result */
-    int flags                      /* FIO_ flags */
+/// Convert a Unicode character to bytes.
+///
+/// @param         c     The character to convert
+/// @param[in,out] pp    The pointer to store the result at
+/// @param         flags The FIO_ flags to use
+///
+/// @return true for an error, false when it's OK.
+static bool ucs2bytes(
+    unsigned c,
+    char_u **pp,
+    int flags
```

Comment:
This doesn't follow the style guide. It should be on same line or aligned on parenthese if too long.